src/util/toggle-class.js

```diff
@@ -0,0 +1,38 @@
+//
+// helper to add/remove classes for IE9
+//
+// Usage:
+//
+// toggleClass(div, 'demo'); // Toggles the class `demo`
+// toggleClass(div, 'demo', true); // Adds the class `demo`
+// toggleClass(div, 'demo', false); // removes the class `demo`
+//
+export default function toggleClass(element, className, toggleClass) {
+  const classes = element.className === '' ? [] : element.className.split(' ');
+  const idx = classes.indexOf(className);
+  const hasClass = idx !== -1;
+  if (toggleClass === undefined) {
+    toggleClass = !hasClass;
+  }
+  // Break if classes are already set/removed
+  if (toggleClass === hasClass) {
+    return;
+  }
+  // Remove class
+  if (!toggleClass) {
+    classes.splice(idx, 1);
+  }
+  // Add class
+  if (toggleClass) {
+    classes.push(className);
+  }
+  element.className = classes.join(' ');
+}
+
+export function removeClass(element, className) {
+  return toggleClass(element, className, false);
+}
+
+export function addClass(element, className) {
+  return toggleClass(element, className, false);
```

Comment:
I assume the force flag should be `true` here? the tests don't cover `addClass` and `removeClass`…